src/flamegraph.cpp

```diff
@@ -403,13 +476,24 @@ SearchResults applySearch(FrameGraphicsItem* item, const QString& searchValue)
 }
 }
 
+void updateFlameGraphColorScheme(FrameGraphicsItem* item)
+{
+    item->setBrush(brush(item->symbol(), BrushType::Hot));
```

Comment:
this looks bogus, don't you want to pass the scheme in as a parameter to this function and then use it here?!